src/components/Avatar.js

```diff
@@ -106,9 +107,9 @@ function Avatar(props) {
             ) : (
                 <View style={[iconStyle, StyleUtils.getAvatarBorderStyle(props.size, props.type), ...props.iconAdditionalStyles]}>
                     <Image
-                        source={{uri: props.source}}
+                        source={{uri: `${props.source}?cacheBuster=${cacheBuster}`}}
```

Comment:
This seems like a solid solution but we shouldn't blindly concatenate the `cacheBuster` param here. The uri could be in the format `image.jpg?width=400` and in this case adding another `?` will not work as expected

Reply:
Let me figure out it.